tests/qconvolutional_test.py

```diff
@@ -201,25 +200,37 @@ def test_sequential_qnetwork():
   return model
 
 
-def test_qconv1d():
+@pytest.mark.parametrize("layer_cls", ["QConv1D", "QSeparableConv1D"])
+def test_qconv1d(layer_cls):
   np.random.seed(33)
-  x = Input((4, 4,))
-  y = QConv1D(
+  if layer_cls == "QConv1D":
+    x = Input((4, 4,))
+    y = QConv1D(
       2, 1,
       kernel_quantizer=quantized_bits(6, 2, 1, alpha=1.0),
       bias_quantizer=quantized_bits(4, 0, 1),
       name='qconv1d')(
           x)
-  model = Model(inputs=x, outputs=y)
+    model = Model(inputs=x, outputs=y)
+  else:
+    x = Input((4, 4,))
+    y = QSeparableConv1D(
+      2, 2,
+      depthwise_quantizer=quantized_bits(6, 2, 1, alpha=1.0),
+      pointwise_quantizer=quantized_bits(4, 0, 1, alpha=1.0),
+      bias_quantizer=quantized_bits(4, 0, 1),
+      name='qsepconv1d')(
+          x)
+    model = Model(inputs=x, outputs=y)
 
   # Extract model operations
   model_ops = extract_model_operations(model)
 
-  # Check the input layer model operation was found correctly
-  assert model_ops['qconv1d']['type'][0] != 'null'
```

Comment:
Why was this check removed?

Reply:
Indeed, this should not have been removed. I see now that the problem is that I forgot to support `QSeparableConv2D` in `extract_model_operations`, and probably removed this line while cleaning up commented code.

Reply:
Oops my bad! Yes, you are right. I must have been looking at the wrong diff